src/tests/ftest/security/pool_connect_init.py

```diff
@@ -0,0 +1,146 @@
+        mode = self.params.get("mode", '/run/poolparams/createmode/', 511)
+        setid = self.params.get("setname", '/run/poolparams/createset/',
+                                "daos_server")
+        size = self.params.get("size", '/run/poolparams/createsize/',
+                               107374182)
+        root_id = 0
+        user_uid = os.geteuid()
+        user_gid = os.getegid()
+        try:
+            self.log.info("===Create DAOS pool1 with:")
+            self.log.info("===uid:  %s", user_uid)
+            self.log.info("===gid:  %s", root_id)
+            self.pool1 = DaosPool(self.context)
+            self.pool1.create(mode, user_uid, root_id, size, setid)
+            self.log.info("===>Pool1 create succeed.")
+            self.log.info("===Create DAOS pool2 with:")
+            self.log.info("===uid:  %s", root_id)
+            self.log.info("===gid:  %s", user_gid)
+            self.pool2 = DaosPool(self.context)
+            self.pool2.create(mode, root_id, user_gid, size, setid)
+            self.log.info("===>Pool2 create succeed.")
+            self.log.info("===Create DAOS pool3 with:")
+            self.log.info("===uid:  %s", root_id)
+            self.log.info("===gid:  %s", root_id)
+            self.pool3 = DaosPool(self.context)
+            self.pool3.create(mode, root_id, root_id, size, setid)
+            self.log.info("===>Pool3 create succeed.")
+        except DaosApiError as exc:
+            self.log.info(exc)
+            self.log.info(traceback.format_exc())
+            self.fail("##(0)Pool create failed in setUp.\n")
+
+    def tearDown(self):
+        try:
+            if self.pool1 is not None:
+                self.pool1.disconnect()
+                self.pool1.destroy(1)
+            if self.pool2 is not None:
+                self.pool2.disconnect()
+                self.pool2.destroy(1)
+            if self.pool3 is not None:
+                self.pool3.destroy(1)
+        except DaosApiError as exc:
+            self.log.info(exc)
+            self.log.info(traceback.format_exc())
+            self.fail("##(2)Pool destroy failed in tearDown.\n")
+        finally:
+            super(PoolSecurityTest, self).tearDown()
+
+    def test_poolconnect(self):
+        """
+        Test basic pool security in pool creation and connect.
+        DAOS-2930: DAOS security: initial tests pool creation with defaul
+                   ACL and connect.
+        Tests DAOS pools creation, and connect with:
+        (1)pool1: match of owner uid.  (succeed)
+        (2)pool2: match of owner gid.  (succeed)
+        (3)pool3: mismatch of owner uid and gid.
+                                       (permission denied with error -1001)
+
+        :avocado: tags=pool,security_basic,security
+        """
+        user_uid = os.geteuid()
+        user_gid = os.getegid()
+        der_no_permission = "RC: -1001"
+        try:
+            self.log.info("===Connecting to the DAOS pool1")
```

Comment:
For clarity, maybe the output should say which case each is, i.e. connecting to default ACL with matching owner, connecting to default ACL with matching owner-group, connecting with no match to default ACL